04-understanding-survey-data-documentation.Rmd

```diff
@@ -218,7 +217,7 @@ Once we've pulled the data, we want to ensure that the data only includes the 50
 ```{r}
 #| label: understand-cps-state
 cps_state %>%
-  count(state)
+  count(state) 
```

Comment:
This outputs a list with state being a number instead of abbreviations.  Do we know of a way to make this abbreviations so we can look at it?  Would be best if we could find a way to easily show the reader this is only the 50 states +DC that doesn't rely on us just saying "yup there are 51 rows in this"

Reply:
This is always my sanity check. Open to other suggestions. Also, is it necessary at all? I think CPS documentation states it's only for the 50 states + DC.